nodejs/week1/warmUp/circleCalculator.js

```diff
@@ -0,0 +1,28 @@
+const x = Math.PI.toFixed(2);
```

Comment:
Appreciate the idea of creating a variable so that you can type less :) 

When you do this, you should provide a descriptive name for the variable so that a person that reads your code can quickly understand what the variable represents. In this case, x could be anything. A better name that would be more descriptive and let a reader understand what the variable stands for would be PI. 

const PI = Math.PI 

the toFixed(2) is tricky. toFixed(2) transforms Math.PI (a number) to a string which means that whenever you try to perform a arithmetic operation between a number and that variable the results will be unexpected. Because of that, you should limit the use of toFixed() for whenever you want to present the data to the user.

If you are curious try evaluating the following expressions:

1 + "1"
"1" + 1

And see the result.